openmls/src/group/public_group/validation.rs

```diff
@@ -107,129 +110,244 @@ impl PublicGroup {
 
     // === Proposals ===
 
-    /// Validate Add proposals. This function implements the following checks:
+    /// Validate key uniqueness. This function implements the following checks:
     ///  - ValSem101
     ///  - ValSem102
+    ///  - ValSem103
     ///  - ValSem104
-    ///  - ValSem106
```

Comment:
While you're at it, could you add the full ValSem string at least up here in the function docs and ideally also for the ValSems below below?